games.rb

```diff
@@ -1,5 +1,53 @@
 class Games 
   def balls(bowling)
-    @bowling = bowling
+    counter_frame = 0
+    ball = 0
+    frame = []
+    score = 0
+    while ball < 19
+      total = game_rules(bowling[ball], bowling[ball + 1], bowling[ball + 2], bowling[ball + 3], bowling[ball + 4], counter_frame)
```

Comment:
> complemento del cuarto comentario

si decides usar hash como parametro para los balls, podrias implementar algo asi:
```suggestion
      balls.each_with_index { |(key, value), index| balls[key] = ball + index } 
      total = game_rules(balls, counter_frame)
```
en caso de un array: 
```suggestion
       balls.map.with_index { |elem, index| balls[index] = ball + index }
      total = game_rules(balls, counter_frame)
```